gensim/corpora/lowcorpus.py

```diff
@@ -23,12 +23,22 @@
 
 
 def split_on_space(s):
+    """
+
+    Parameters
+    ----------
+    s :
```

Comment:
???? (empty descriptions here and everywhere)

Reply:
>Docs formally comply with numpy style now but not all type annotations and descriptions are there.

:)